server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreIT.java

```diff
@@ -120,6 +122,16 @@ public void testPeerRecoveryWithRemoteStoreAndRemoteTranslogFlush() throws Excep
         testPeerRecovery(randomIntBetween(2, 5), true);
     }
 
+    public void testPeerRecoveryWithLowActivityTimeout() throws Exception {
+        ClusterUpdateSettingsRequest req = new ClusterUpdateSettingsRequest().persistentSettings(
+            Settings.builder()
+                .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey(), "20kb")
+                .put(RecoverySettings.INDICES_RECOVERY_ACTIVITY_TIMEOUT_SETTING.getKey(), "1s")
+        );
+        internalCluster().client().admin().cluster().updateSettings(req).get();
+        testPeerRecovery(randomIntBetween(2, 5), true);
```

Comment:
With `1s` timeout, recovery should fail, right? Is `testPeerRecovery` checking it?

Reply:
The default value of the recovery timeout setting is 30mins which means that if there is any issue with peer recovery it would wait until 30th minute for the peer recovery to fail. With this we have reduced the recovery timeout and recovery bytes per sec to limit where without this code change, this test is flaky (recovery fails some times).  With the code change, the test is passing. I have ran it on a loop. I will update here if I see any failure.